code/__defines/_macros.dm

```diff
@@ -78,5 +78,5 @@
 #define CanInteract(user, state) (CanUseTopic(user, state) == STATUS_INTERACTIVE)
 
 #define isopenturf(target) istype(target, /turf/simulated/open)
-#define isweakref(target) istype(target, /datum/weakref)
+#define isweakref(target) istype(target, /weakref)
```

Comment:
Aren't these two equal? At which point, I'd rather have the ref declared as an explicit child of datum.